examples/lusid/orders/blocks/custom_button/2-create-custom-button.sql

```diff
@@ -0,0 +1,62 @@
+@@providerName = select 'Set_Contingent_Order_ID_KR_five';
```

Comment:
Description: Configure a custom button in the LUSID Order Blotter to set contingent ids on the selected Blocks